lib/fathead/mdn_css/parse.pl

```diff
@@ -431,6 +439,22 @@ sub parse_fragment_data {
     }
 }
 
+sub parse_initial_value {
+    my ($table_properties) = @_;
+
+    #first <tr> Contains the initial value
+    my $tr = $table_properties->find('tr')->first;
+    my $th = $tr->at('th');
+    my $initial_value;
+    if ($th) {
+        my $a = $th->at('a');
+        if ( $a && $a->text =~ /Initial value/ ) {
+            $initial_value = $tr->all_text;
```

Comment:
Looks like this needs a little cleaning up, there's no space between "Initial Value" and the actual value